toolkit/tools/internal/packagerepo/repomanager/rpmrepomanager/rpmrepomanager.go

```diff
@@ -16,12 +16,31 @@ import (
 
 // CreateRepo will create an RPM repository at repoDir
 func CreateRepo(repoDir string) (err error) {
+	logger.Log.Debugf("Creating RPM repository in (%s)", repoDir)
+
+	err = CleanupRepo(repoDir)
+
+	if err != nil {
```

Comment:
Nit: usually we don't add an empty line between a function call and the check for the returned `err`.